.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Testing
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - uses: actions/setup-node@v1
```

Comment:
Can we pin a Node version with `nodenv`?

Reply:
👍 Done!